src/plugins/vis_type_timeseries/common/timefield_utils.ts

```diff
@@ -17,12 +17,14 @@
  * under the License.
  */
 
-import _ from 'lodash';
+import { TimeFieldObject } from './types';
 
-export const createSelectHandler = (handleChange) => {
-  return (name) => (selectedOptions) => {
-    return handleChange?.({
-      [name]: _.get(selectedOptions, '[0].value', null),
-    });
-  };
+const isTimefieldObject = (timeField: any): timeField is { name: string; label: string } => {
```

Comment:
Can we avoid this `any here`?

Reply:
I think 'any' is ok for type guard functions,  but ok, let's specify type.